intg-denonavr/media_player.py

```diff
@@ -11,7 +11,7 @@
 import avr
 from config import AvrDevice, create_entity_id
 from ucapi import EntityTypes, MediaPlayer, StatusCodes
-from ucapi.media_player import Attributes, Commands, DeviceClasses, Features, States
+from ucapi.media_player import Attributes, Commands, DeviceClasses, Features, States, Options
```

Comment:
isort lint error: Imports are incorrectly sorted and/or formatted.
Fix with `python -m isort intg-denonavr/.`

Reply:
fixed